lib/utils/keymaps.js

```diff
@@ -1,4 +1,5 @@
 import {remote} from 'electron';
+import {returnCommand} from './keymaps-normalize';
 
 const {getKeymaps} = remote.require('./config');
```

Comment:
If we move `keymaps-normalize` into `app` directory, we can require only a `getCommand` function.
`getCommand(keys)`.
`getKeymaps()`will be called directly in this `getCommand` function, on app side.

Reply:
Nice idea, I'll look into it. :)